_schedule/talks/2021-10-22-bootstrapping-your-local-python-environment.md

```diff
@@ -0,0 +1,64 @@
+---
+abstract: Let’s talk about getting started with the end in mind and making sure your
+  development computer doesn’t become the next superfund site. We’ll quickly go through
+  a tour of options such as pyenv, venv, virtualenv, conda and Docker as great ways
+  to make sure you can develop in a sane environment.
+accepted: true
+category: talks
+date: 2021-10-22 10:00
+difficulty: All
+layout: session-details
+permalink: /talks/bootstrapping-your-local-python-environment/
+presenters:
+- bio: "Calvin Hendryx-Parker is the co-founder and CTO of Six Feet Up, a software
+    company that helps organizations build apps faster, innovate with AI, simplify
+    Big Data, and leverage Cloud technology. At Six Feet Up, Calvin establishes the
+    company's technical vision and leads all aspects of the company's technology development.
+    He provides the strategic vision for enhancing the offerings of the company and
+    infrastructure, and he works with the team to set company priorities and implement
+    processes that will help improve product and service development. \r\n\r\n In
+    2019, Calvin was named an AWS Community Hero — one of only 23 Heroes in North
+    America. He is the founder and host of the Python Web Conference, the largest
+    worldwide event for web developers; the co-founder of IndyPy, the largest Python
+    meetup in Indiana with 1,900+ members; and the founder of IndyAWS, the fastest
+    growing cloud meetup in the state with 800+ members. Additionally, Calvin is the
+    driving force behind LoudSwarm by Six Feet Up, a high impact virtual event platform
+    that debuted in June 2020.\r\n\r\nOutside of work, Calvin spends time tinkering
```

Comment:
@jefftriplett will jekyll properly render the escaped newlines or do we need to edit that out?

Reply:
I honestly don't remember. I'll experiment with Calvins since I think we have to mesh up the speaker pages to be able to see them. 

It should eventually be here: https://deploy-preview-45--djangoconus2021.netlify.app/talks/bootstrapping-your-local-python-environment/

Reply:
Right now the bios are empty, so I'm assuming that we need to create the speaker files before these will be visible.

Reply:
I updated the script we use for fixing these. I'll merge in your changes then run it.